.github/workflows/flambda-backend.yml

```diff
@@ -102,4 +102,9 @@ jobs:
       working-directory: merlin-jst
       run: |
         export MERLIN_TEST_OCAML_PATH=$GITHUB_WORKSPACE/flambda-backend/_install
+        opam exec -- dune build
         opam exec -- dune runtest -p merlin-lib,dot-merlin-reader,merlin,ocaml-index
+        # Check if dune promoted any files. If this check fails, the likely culprit is
+        # that you compiled with the wrong menhir version. Cross-check your opam switch's
+        # menhir version with the one in merlin-lib.opam
+        git diff --quiet
```

Comment:
If this fails, how would we be directed to look at the helpful comment just above?

Reply:
done